tests/test_users.py

```diff
@@ -132,18 +132,25 @@ async def test_login():
             },
         )
 
-        cookie = res.cookies.get("fastapiusersauth")
+        token = res.cookies.get(settings.access_token_name)
+
         payload = jwt.decode(
-            cookie,
-            settings.secret_key,
+            token,
+            settings.access_token_secret_key,
             algorithms=settings.algorithm,
-            audience="fastapi-users:auth",
+            audience=settings.token_audience,
         )
         user_id = payload["sub"]
 
         assert user_id == str(login_user.id)
         assert res.status_code == SUCCESS_LOGIN_STATUS_CODE
 
+        response = await make_http_request(
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Consider adding assertions for the response content.

While the status code check is good, verifying the response content ensures the correct user data is returned.

```suggestion
        response = await make_http_request(
            "/api/users/me", method=RequestMethod.GET, cookies=res.cookies
        )

        assert response.status_code == 200
        response_data = response.json()
        assert response_data["id"] == str(login_user.id)
        assert response_data["username"] == login_user.username

@pytest.mark.parametrize(
    "username, password, status_code",
```